homeassistant/components/ccl/coordinator.py

```diff
@@ -0,0 +1,45 @@
+"""The CCL Data Update Coordinator."""
+
+from __future__ import annotations
+
+import logging
+
+from aioccl import CCLDevice, CCLSensor
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class CCLCoordinator(DataUpdateCoordinator):
```

Comment:
```suggestion
class CCLCoordinator(DataUpdateCoordinator[..]):
```
Let's add the return type of `async_update_data` in the typeslot